ui/public/checkBrowser.js

```diff
@@ -0,0 +1,27 @@
+function checkBrowser() {
+  if (!window.__supported_browsers__.test(navigator.userAgent)) {
+    var text
+    if (navigator.language.indexOf('zh') === 0) {
+      text =
+        '您的浏览器版本已过期，推荐使用最新版本的 Chrome 或 Edge 浏览器以便获得更好的体验。'
```

Comment:
UTF8 characters might cause problems in some old browsers I guess, please test it against IE6

Reply:
Really, just know it, 😂 Where I can find an IE6 now... anyway, I will try it, maybe I can test it in IE11 first. (ant design only support IE11 for IEs)

Reply:
A note.. AFAIK TiDB Dashboard won't work on IE11 as well

Reply:
IE6 has a wrong charset detecting strategy https://support.microsoft.com/zh-cn/help/928847/internet-explorer-uses-the-wrong-character-set-when-it-renders-an-html